### ISSUE TITLE
Keyboard support still needs work

### ISSUE THREAD

**Issue by sandersky** [ciena-frost/ember-frost-date-picker#24]
A keyboard user should be able to do more than control the text input for time. Once I tab onto the time input and the widget pops up I should be able to tab into the hour/minute/second labels in the widget and arrow up/down. Here is an example of something I did a few years ago that is a bit more keyboard friendly which may serve as a good example: http://www.mattdahl.io/bootstrap-datetime/.

> Note: my date/time widget is far from perfect.

I'm also not a fan of the widgets auto-opening when I tab onto the inputs as a keyboard user sometimes wants to tab on past the fancy controls (but when the controls are open would be tabbing through them).
